src/main/java/com/minecolonies/coremod/util/WorkerUtil.java

```diff
@@ -165,11 +168,11 @@ public static boolean setSpawnPoint(@Nullable final BlockPos spawnPoint, @NotNul
      * @param blockHardness the hardness.
      * @return the toolType to use.
      */
-    public static IToolType getBestToolForBlock(final BlockState state, float blockHardness)
+    public static IToolType getBestToolForBlock(final BlockState state, float blockHardness, final AbstractBuilding building)
     {
         final net.minecraftforge.common.ToolType forgeTool = state.getHarvestTool();
 
-        if (state.getBlock() instanceof IForgeShearable)
+        if (building.getOptionalSetting(USE_SHEARS).orElse(new BoolSetting(false)).getValue() && state.getBlock() instanceof IForgeShearable)
```

Comment:
Looks fine, only thing I'd change would be default to true.

Reply:
done